Library/Formula/xpwn.rb

```diff
@@ -0,0 +1,32 @@
+class Xpwn < Formula
+  homepage "https://github.com/planetbeing/xpwn"
+  url "https://github.com/planetbeing/xpwn.git"
```

Comment:
Homebrew/homebrew doesn't accept `.git` URLs as the main download. It's impossible to bottle, notice when updated, etc. If there's no stable download this will need to go in ` Homebrew/head-only`.